src/components/Home/Home.css

```diff
@@ -0,0 +1,17 @@
+.center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
I think it will be better to seperate out those classes which can be used at multiple place like align-item, cursor-pointer, display, justify_content and placed them in new file. This way we can use these classed in views instead of making the css bulky